build.sbt

```diff
@@ -34,15 +45,17 @@ ThisBuild / githubWorkflowPublish := Seq(
   )
 )
 
-addCommandAlias("validateJVM", ";clean;compile;test")
-
-lazy val modules: List[ProjectReference] = List(
-  deriving,
-  test,
-  typeable
-)
+addCommandAlias("validate", ";clean;validateJVM;validateJS")
+addCommandAlias("validateJVM", ";buildJVM;testJVM")
+addCommandAlias("validateJS", ";buildJS;testJS")
+addCommandAlias("buildJVM", ";derivingJVM/compile;testJVM/compile;typeableJVM/compile")
+addCommandAlias("buildJS", ";derivingJS/compile;testJS/compile;typeableJS/compile")
+addCommandAlias("testJVM", ";derivingJVM/test;testJVM/test;typeableJVM/test")
+addCommandAlias("testJS", ";derivingJS/test;testJS/test;typeableJS/test")
```

Comment:
Doesn't this force sequential execution on the subtasks of the command? I prefer using `all derivingJVM/compile testJVM/compile typeableJVM/compile` but that doesn't compose so I'm not sure.

Reply:
I'm actually fairly happy having these serialized.